mobile.go

```diff
@@ -104,7 +105,10 @@ func doUpdateMobile(url string, out *os.File, updater Updater) error {
 	}
 
 	req.Header.Add("Accept-Encoding", "gzip")
-	pro.PrepareForFronting(req)
+
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
```

Comment:
So looking at this deeper a few things jump out here. Again, I now remember code reviewing this the first time around, so my apologies for completely botching that somehow! I think we should try to go in and clean this up a little bit while we're here, which I think would include:

1. Not using the `pro` package at all
2. I think also not using that general pro approach here of using a separate `RoundTripper` for GET/HEAD and other requests. For updates all requests (including the POST) are idempotent (which is not the case for the pro server), so I think we can just safely use `proxied.ParallelPreferChained` and can skip both calls to `pro.GetHTTPClient()` in this file.

I think that's really it. Ideally we might also re-use the http client just because it will remember things like if the chained proxy worked the last time, but that's not a biggie.